commons-statistics-descriptive/src/test/java/org/apache/commons/statistics/descriptive/TestHelper.java

```diff
@@ -48,14 +48,14 @@ static double[] concatenate(double[]... arrays) {
     /**
      * Helper function to compute the expected value of Mean using BigDecimal.
      * @param values Values.
-     * @return Mean of values.
+     * @return Mean of values rounded to <a href = "https://en.wikipedia.org/wiki/Decimal128_floating-point_format"> DECIMAL128 precision.
```

Comment:
The `<a>` tag looks malformed to me, but if that renders valid Javadoc & HTML then I guess that's OK.

Reply:
It opens a webpage in my IDE so the href is OK. But it should have a `</a>` tag. Note that testing with:
```
mvn javadoc:test-javadoc
```
errors as there are no public or protected test classes. So this link is just for internal use only.